include/tc/core/execution_engine.h

```diff
@@ -106,7 +106,7 @@ class ExecutionEngine {
   size_t emplaceExecutor(std::unique_ptr<ExecutorInfo> p);
 
   /// For thread-safety perform all cheap operations under lock.
-  std::mutex executorInfoMutex;
+  std::mutex executorInfoMutex_;
```

Comment:
should we update the coding conventions to say "private _and protected_ members" should be suffixed with underscore https://github.com/facebookresearch/TensorComprehensions/blob/master/CodingConventions.md#class-data-members ?